src/components/votingcenter/CenterListItem.js

```diff
@@ -1,12 +1,28 @@
-import React from "react";
+import React, {useContext} from "react";
+import { AppContext } from "../app/App";
+import { useHistory } from "react-router-dom";
+
+const CenterListItem = (props) => {
+  const {
+    id,
+    address,
+    pollingHours,
+    startDate,
+    endDate,
+    typeOfVote,
+  } = props;
+
+  const history = useHistory();
+  const { dispatch } = useContext(AppContext);
+
+  const handleClick = (props) => {    
+    dispatch({
+      type: "changeSelectedCenter",
+      payload: props
+    });
+    history.push(`/voting-centers/${id}`);
+  }
```

Comment:
after dispatching the action, it redirects to the voting center detail page

Reply:
also sending the entire prop object for now since I'm not sure how much of the data we're going to display on the details page

Reply:
Most excellent! This is great. I didn't realize you could use history.push like this. Very cool.